eng/common/templates/job/source-build.yml

```diff
@@ -48,11 +48,11 @@ jobs:
     pool:
       ${{ if eq(variables['System.TeamProject'], 'public') }}:
         name: $[replace(replace(eq(contains(coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranch'], 'refs/heads/main'), 'release'), 'true'), True, 'NetCore-Svc-Public' ), False, 'NetCore-Public')]
-        demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
```

Comment:
This is eng/common - this change needs to happen in arcade.

Reply:
I've a quick question. How do you mean "in arcade"? The arcade repo?

Reply:
Yes, anything in eng/common is updated by arcade on each update from that repo, so it will always clobber this change.

Reply:
To be clear, we DON'T have to get all the 1804 instances in this PR. We can do a update PR. We do, however, need to make this PR run clean (obviously).

Reply:
Got it. Since we are fine with gradually upgrading our lab machines, I can restore this specific change back to 18.04 and in the meantime, I'll work on the arcade part

Reply:
Sorry. I should have mentioned that initially.